dashboard/app/controllers/api/v1/pd/regional_partner_program_registrations_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Api::V1::Pd::RegionalPartnerProgramRegistrationsController < ApplicationController
+  authorize_resource class: 'Pd::RegionalPartnerProgramRegistration', only: :create
+
+  def create
+    form_data_hash = params.try(:[], :form_data) || {}
+    form_data_json = form_data_hash.to_unsafe_h.to_json.strip_utf8mb4
+
+    teachercon = params.try(:[], :teachercon)
```

Comment:
Params should always be present in the controller actions. You don't need `try(:[]`...